cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/config/ETLConfig.java

```diff
@@ -14,23 +14,22 @@
  * the License.
  */
 
-package co.cask.cdap.templates.etl.batch;
+package co.cask.cdap.templates.etl.api.config;
 
-import co.cask.cdap.templates.etl.common.config.ETLStage;
 import com.google.common.base.Objects;
 
 import java.util.List;
 
 /**
- * ETL Batch Adapter Configuration.
+ * ETL Adapter Configuration.
  */
-public class ETLBatchConfig {
+public final class ETLConfig {
```

Comment:
Why the class name change to ETLConfig? Since we have schedule as a field, wouldn't this be specific to a Batch adapter?